### PR TITLE
DM-31825: Implement RFC-807 on DRP parquet table consistency

### PR DESCRIPTION
**Pull request by yalsayyad**
* Added all the columns used by pipe_analysis and analysis_drp to the object table
* Added all the apertures to the aperture fluxes
*  all dataframes have primary keys as index only
* object table columns look like `g_apFlux` instead of `gApFlux`
* All per-patch datasets are consolidated into a per-tract dataset
* names start with lowercase

I promise to squash these before merging

### CODE REVIEW

File: policy/imsim/ForcedSource.yaml [hunk 1]
```diff
@@ -21,67 +25,58 @@ funcs:
...
+    psfDiffFluxErr:
         functor: LocalNanojanskyErr
         dataset: diff
         args:
             - base_PsfFlux_instFlux
             - base_PsfFlux_instFluxErr
             - base_LocalPhotoCalib
             - base_LocalPhotoCalibErr
-    PsDiffFluxFlag:
+    psfDiffFluxFlag:
```
Comment (sr525) [inline policy/imsim/ForcedSource.yaml:64]: Should this be Flag or _flag?

File: policy/imsim/Object.yaml [hunk 2]
```diff
@@ -254,45 +234,138 @@ funcs:
...
+    ap03FluxErr:
         functor: NanoJanskyErr
         dataset: forced_src
         args:
-            - base_CircularApertureFlux_25_0_instFlux
-            - base_CircularApertureFlux_25_0_instFluxErr
-    Ap9Flux:
+            - base_CircularApertureFlux_3_0_instFlux
+            - base_CircularApertureFlux_3_0_instFluxErr
+    ap03Flux_flag:
```
Comment (sr525) [inline policy/imsim/Object.yaml:319]: _flag or Flag.

File: policy/imsim/Object.yaml [hunk 3]
```diff
@@ -254,45 +234,138 @@ funcs:
...
+        functor: Column
+        dataset: forced_src
+        args: base_CircularApertureFlux_3_0_flag
+    ap06Flux:
+        functor: NanoJansky
+        dataset: forced_src
+        args: base_CircularApertureFlux_6_0_instFlux
+    ap06FluxErr:
```
Comment (sr525) [inline policy/imsim/Object.yaml:327]: There is an inconsistency between the gaap and the ap flux naming conventions. The ap flux ones have no underscores.

File: policy/imsim/Object.yaml [hunk 4]
```diff
@@ -93,131 +93,111 @@ funcs:
...
-        args: slot_ModelFlux
-        filt1: z
-        filt2: y
+    # grStd:
+    # riStd:
+    # izStd:
+    # zyStd:
     #  PSF GAaP flux
-    GaapFlux_Psf:
+    gaapFlux_Psf:
```
Comment (sr525) [inline policy/imsim/Object.yaml:151]: Psf -> psf

File: policy/imsim/Object.yaml [hunk 5]
```diff
@@ -93,131 +93,111 @@ funcs:
...
         functor: NanoJansky
         dataset: forced_src
         args: ext_gaap_GaapFlux_1_15x_PsfFlux_instFlux
-    GaapFluxErr_Psf:
+    gaapFluxErr_Psf:
```
Comment (sr525) [inline policy/imsim/Object.yaml:155]: Psf -> psf

File: policy/imsim/Object.yaml [hunk 6]
```diff
@@ -93,131 +93,111 @@ funcs:
...
-    GaapFluxErr_1_0:
+    gaapFluxErr_1_0:
         functor: NanoJanskyErr
         dataset: forced_src
         args:
             - ext_gaap_GaapFlux_1_15x_1_0_instFlux
             - ext_gaap_GaapFlux_1_15x_1_0_instFluxErr
     #  Optimal GAaP flux
-    GaapFlux_Optimal:
+    gaapFlux_Optimal:
```
Comment (sr525) [inline policy/imsim/Object.yaml:184]: Optimal -> optimal

File: policy/imsim/Object.yaml [hunk 7]
```diff
@@ -93,131 +93,111 @@ funcs:
...
         functor: NanoJansky
         dataset: forced_src
         args: ext_gaap_GaapFlux_1_15x_Optimal_instFlux
-    GaapFluxErr_Optimal:
+    gaapFluxErr_Optimal:
```
Comment (sr525) [inline policy/imsim/Object.yaml:188]: Optimal -> optimal

Comment (arunkannawadi) [reply policy/imsim/Object.yaml:188]: For consistency with aperture fluxes, this should become gaapOptimalFlux(Err). I'll push a commit in DM-31912 that can be cherry-picked and included here.

Comment (arunkannawadi) [reply policy/imsim/Object.yaml:188]: And here's that commit: https://github.com/lsst/obs_lsst/pull/352/commits/4f613f60c327583c07840f35c573137d608deeff

File: policy/imsim/Object.yaml (hunk 4 above)
Comment (sr525) [reply policy/imsim/Object.yaml:151]: Arun just changed these to gaapPsfFlux.

File: policy/imsim/Object.yaml (hunk 5 above)
Comment (sr525) [reply policy/imsim/Object.yaml:155]: gaapPsfFluxErr

File: policy/imsim/Object.yaml (hunk 6 above)
Comment (sr525) [reply policy/imsim/Object.yaml:184]: gaapOptimalFlux

File: policy/imsim/Object.yaml (hunk 7 above)
Comment (yalsayyad) [reply policy/imsim/Object.yaml]: If you're changing it on your ticket, I'm not going to bother here.

**Pull request merged**